detail/node.vue

```diff
@@ -81,65 +109,95 @@ export default {
       return this.mapToStatus(this.value.isKubeletOk);
     },
 
-    statusTableRows() {
+    conditionsTableRows() {
       return this.value.status.conditions;
     },
 
-    systemTableRows() {
+    infoTableRows() {
       return Object.keys(this.value.status.nodeInfo)
         .map(key => ({
-          key,
+          key:   capitalize(words(key).join(' ')),
           value: this.value.status.nodeInfo[key]
         }));
     },
 
-    labelsTableRows() {
-      return Object.keys(this.value.metadata.labels)
-        .map(key => ({
-          key,
-          value: this.value.metadata.labels[key]
-        }));
+    addressTableRows() {
+      return this.value.status.addresses;
     },
 
-    annotationsTableRows() {
-      return Object.keys(this.value.metadata.annotations)
-        .map(key => ({
-          key,
-          value: this.value.metadata.annotations[key]
-        }));
+    imageTableRows() {
+      return this.value.status.images.map(image => ({
+        name:      image.names[1],
+        sizeBytes: image.sizeBytes
+      }));
+    },
+
+    taintTableRows() {
+      return this.value.spec.taints || [];
     },
 
     detailTopColumns() {
       return [
         {
-          title: 'IP Address',
-          name:  'ipAddress'
+          title: 'State',
+          name:  'state'
+        },
+        {
+          title:   'IP Address',
+          content:  this.value.internalIp
         },
         {
-          title: 'Version',
-          name:  'version'
+          title:   'Version',
+          content:  this.value.version
         },
         {
-          title: 'OS',
-          name:  'os'
+          title:   'OS',
+          content:  this.value.status.nodeInfo.osImage
         },
         {
-          title: 'Created',
-          name:  'created'
+          title:   'Container Runtime',
+          name:    'container-runtime'
         },
       ];
     }
   },
 
+  mounted() {
+    this.metricPoller.start();
+  },
+
+  beforeDestroy() {
+    this.metricPoller.stop();
+  },
+
   methods: {
     memoryFormatter(value) {
-      return (value / 1048576).toFixed(2);
+      const formatOptions = {
+        maxExponent: exponentNeeded(this.value.ramCapacity),
+        minExponent: exponentNeeded(this.value.ramCapacity),
```

Comment:
I actually needed to completely remove the suffix and compute the suffix so that it worked with the consumption gauges better.

I did make the spacing update you requested though. I was actually really curious why we had the spacing we did. I thought I was just missing something.